doc/releasenotes/14_0_0_summary.md

```diff
@@ -268,4 +313,7 @@ path is to upgrade vtctld, run `SetKeyspaceDurabilityPolicy` and then upgrade VT
 Work has gone into making the advisory locks (`get_lock()`, `release_lock()`, etc.) release reserved connections faster and in more situations than before.
 
 ### Pre-Legacy Resharding is now deprecated
-A long time ago, the sharding column and type were specified at the keyspace level. This syntax is now deprecated and will be removed in v15.
\ No newline at end of file
+A long time ago, the sharding column and type were specified at the keyspace level. This syntax is now deprecated and will be removed in v15.
+
+### Drop the use of the legacy healthcheck in VTCtld
+In release `7.0.0`, a new healthcheck was developed and the old one was renamed legacy healthcheck. In Vitess 14, we have changed VTCtld to use the new healthcheck instead of the legacy.
```

Comment:
I'm not sure we really need this, it feels like an implementation detail not a user facing feature

Reply:
Good point, I removed it.